packages/shared/src/components/modals/streaks/StreakRecoverModal.tsx

```diff
@@ -0,0 +1,226 @@
+        fill="#FC538D"
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M68.5 67.5C82.5833 67.5 94 56.0833 94 42C94 27.9167 82.5833 16.5 68.5 16.5C54.4167 16.5 43 27.9167 43 42C43 56.0833 54.4167 67.5 68.5 67.5ZM70.8164 27.8369L69.5354 26.1988C69.0182 25.5255 68.068 25.4107 67.4131 25.9423C67.2986 26.0353 67.1977 26.1448 67.1138 26.2674C64.7316 29.7408 65.0171 33.9414 66.674 37.0808L67.3792 38.4291L67.9653 39.576L68.4322 40.5215L68.6773 41.04L68.8694 41.469L69.0251 41.8916C69.4328 43.1655 69.309 44.4867 68.678 45.6046C67.7245 47.2597 66.1742 47.651 64.7168 47.3998C62.3313 46.9885 61.6084 44.2696 61.3894 41.137L61.3429 40.3117L61.3147 39.4751L61.2967 38.2154L61.2942 36.9722L61.2866 36.6824C61.2065 35.2264 60.4855 35.2091 59.1236 36.6305C55.3073 40.6151 54.6646 46.7211 57.5241 51.8778C59.6726 55.8099 64.0219 58.3933 68.472 58.3937C68.6605 58.3937 68.8512 58.3889 69.0456 58.3791C73.6487 58.147 78.089 55.0977 80.0599 50.8149C81.1512 48.4775 81.5161 45.7788 81.0543 43.236C80.4539 39.9303 78.7257 37.7404 76.4219 34.8212C76.1519 34.479 75.874 34.1269 75.5891 33.7618L74.4705 32.3528L72.6891 30.1593L70.8164 27.8369Z"
+        fill="#FC538D"
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M9.5 67.5C23.5833 67.5 35 56.0833 35 42C35 27.9167 23.5833 16.5 9.5 16.5C-4.58326 16.5 -16 27.9167 -16 42C-16 56.0833 -4.58326 67.5 9.5 67.5ZM11.8164 27.8369L10.5354 26.1988C10.0182 25.5255 9.06804 25.4107 8.41308 25.9423C8.29856 26.0353 8.19774 26.1448 8.11376 26.2674C5.73163 29.7408 6.01715 33.9414 7.67405 37.0808L8.37921 38.4291L8.96525 39.576L9.43217 40.5215L9.67726 41.04L9.86942 41.469L10.0251 41.8916C10.4328 43.1655 10.309 44.4867 9.67804 45.6046C8.7245 47.2597 7.17421 47.651 5.71682 47.3998C3.33132 46.9885 2.60845 44.2696 2.38939 41.137L2.34292 40.3117L2.31473 39.4751L2.29672 38.2154L2.29415 36.9722L2.28664 36.6824C2.20653 35.2264 1.48552 35.2091 0.123611 36.6305C-3.69266 40.6151 -4.33545 46.7211 -1.47593 51.8778C0.672583 55.8099 5.02188 58.3933 9.47198 58.3937C9.66052 58.3937 9.85124 58.3889 10.0456 58.3791C14.6487 58.147 19.089 55.0977 21.0599 50.8149C22.1512 48.4775 22.5161 45.7788 22.0543 43.236C21.4539 39.9303 19.7257 37.7404 17.4219 34.8212C17.1519 34.479 16.874 34.1269 16.5891 33.7618L15.4705 32.3528L13.6891 30.1593L11.8164 27.8369Z"
+        fill="#FC538D"
+      />
+    </svg>
+  </div>
+);
+
+const StreakRecoverHeading = ({ days }: { days: number }) => (
+  <Typography
+    className="-mx-1 -mb-1 text-center font-bold"
+    tag={TypographyTag.H3}
+    type={TypographyType.Title1}
+    data-testid="streak-recover-modal-heading"
+  >
+    Oh no! Your
+    <span className="text-accent-bacon-default"> {days} day streak </span> has
+    been broken!
+  </Typography>
+);
+
+const StreakRecoveryCopy = ({
```

Comment:
Would this better be StreakRecoveryBody or sth like it

Reply:
I think all these modal components can be set in their own files

Reply:
I didn't moved those files because the component is still below 250 lines of code (even with the SVG). 
I divided it in component to make easy to find out the business logic, is this confusing?

Reply:
I love the split in components, just thought maybe would be better readable in separate files. But don't have a very strong opinion on this either, some people prefer to not move between files rather have components in the same file since it's just components used here. 

So I'll leave this up to u